runway/data_types.py

```diff
@@ -531,10 +531,10 @@ def __init__(self, description=None, label_to_id=None, label_to_color=None, defa
             raise MissingArgumentError('label_to_id')
         if type(label_to_id) is not dict or len(label_to_id.keys()) == 0:
             msg = 'label_to_id argument has invalid type'
-            raise InvalidArgumentError(msg)
+            raise InvalidArgumentError(self.name or self.type, msg)
```

Comment:
This might be beyond the scope of this PR, but maybe we should auto-set `self.name` to `self.type` in `BaseType` (or have a convenience method `display_name` that returns `self.name or self.type`) so that we don't repeat this pattern?

Reply:
Yeah I'd be down with that. This is a frustrating and error prone pattern to repeat. Weren't we also planning on deprecating name as that now comes from the dictionary key itself instead of the data type?

Reply:
We did deprecate its use as a keyword argument, we're now setting it during the command declaration. I think it's useful for the data type object to have access to its name for referencing in exception messages. What do you think?

Reply:
Ah yeah that's right! I agree so long as we are setting it internally and not relying on the user to set it 👍 

A new PR unifying this display name pattern is in order.

Reply:
Sounds great!